src/vs/workbench/contrib/notebook/browser/view/cellParts/cellOutput.ts

```diff
@@ -183,6 +183,7 @@ class CellOutputElement extends Disposable {
 		const index = this.viewCell.outputsViewModels.indexOf(this.output);
 
 		if (this.viewCell.isOutputCollapsed || !this.notebookEditor.hasModel()) {
+			this.cellOutputContainer.unrenderedChanges();
```

Comment:
Nit: method name could be `flagAsStale()`

Reply:
I was trying to think of a better name for quite a while, thanks